api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentServiceTest.java

```diff
@@ -68,6 +70,9 @@
     @MockBean
     private PaymentCalculator paymentCalculator;
 
+    @SpyBean
```

Comment:
A MockBean here is sufficient as you don't want/need the code to actually be called.

Reply:
Replaced with a MockBean.